### DISCUSSION TITLE
Does the hook "woocommerce_checkout_update_order_meta" still fire ?

### DISCUSSION THREAD

**Discussion by Sartoric** [woocommerce/woocommerce#51086] [Checkout Flow]
I'm facing a strange issue on a clean installation WP 6.6.1 and WC 9.2.3
When I place an order the hook woocommerce_checkout_update_order_meta doesn't fire anymore.

Could it be a bug since I can't find anything in the changelog ? 

Anyone else ? 
TY

**Comment by senadir** [comment]
Yes it fires for shortcode Checkout, for Checkout block, `woocommerce_store_api_checkout_update_order_meta` fires with a different payload of `WC_Order $order` instead of `int $order_id`.

**Reply by Sartoric** [reply]
Hi @senadir 
Thanks for the clarification

Is this pinpointed somewhere in the changelog ? Because I guess I've missed it.

**Reply by alexflorisca** [reply]
The same `woocommerce_checkout_update_order_meta` wasn't implemented in the blocks version of the checkout. The `__experimental_woocommerce_blocks_checkout_update_order_meta` was introduced in https://github.com/woocommerce/woocommerce-blocks/pull/3686. It was then renamed to `woocommerce_store_api_checkout_update_order_meta` in https://github.com/woocommerce/woocommerce-blocks/pull/5992
